OrbitGl/GraphTrack.cpp

```diff
@@ -32,15 +33,32 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   }
 
   float track_z = GlCanvas::kZValueTrack;
-  float text_z = GlCanvas::kZValueText;
 
   Box box(pos_, Vec2(size_[0], -size_[1]), track_z);
 
   // Draw label with graph value at current mouse position.
-  const Color kLabelTextColor(255, 255, 255, 255);
+  const Color kWhiteColor(255, 255, 255, 255);
+  const Color kBlackColor(0, 0, 0, 255);
   double graph_value = GetValueAtTime(time_graph_->GetCurrentMouseTimeNs());
-  canvas->GetTextRenderer().AddText(std::to_string(graph_value).c_str(), canvas->GetMouseX(), y0,
-                                    text_z, kLabelTextColor);
+
+  int white_text_box_width;
+  canvas->GetTextRenderer().GetStringWidth(std::to_string(graph_value).c_str(),
+                                           white_text_box_width);
+
+  Vec2 white_text_box_size(white_text_box_width, layout.GetTextBoxHeight());
```

Comment:
Have you checked if the entire text is rendered when using this size? I had to use a slightly larger size as the last character wasn't rendered when I used the size returned by GetStringSize.

(Could be a mistake I made, but maybe worth double-checking.)

Reply:
You are right, I've just modified the GetString to return the ceiling of the float instead of casting directly to an int. Let me know if it works for you or you still have to add the "+1".

Reply:
That sounds good. I'll try if that works for me once your PR is in.